internal/pkg/component/github/github.go

```diff
@@ -23,14 +25,14 @@ import (
 //TODO: doc about only supporting GitHub with the installed GitHub App
 
 var (
-	ghAppInstallationsCache          *Cache
-	ghAppInstallationsCacheMutex     sync.Mutex
-	ghAppInstallationsCacheID        int64
-	ghAppInstallationTokenCache      *Cache
-	ghAppInstallationTokenCacheMutex sync.Mutex
-	ghAppInstallationTokenCacheID    int64
-	ghAppID                          int64
-	ghAppPrivateKey                  []byte
+	ghAppInstallationsCache              *Cache
+	ghAppInstallationsCacheMutex         sync.Mutex
+	ghAppInstallationsCacheID            int64
+	ghAppInstallationTokenCache          TokenCache
+	ghAppInstallationTokenTimeThreshold  = 30 * time.Minute
+	ghAppInstallationTokenValidityWindow = time.Hour
```

Comment:
Just use `time.hour` is easier than the long name, it can never been changed, so unnecessary to define a variable.